libraries/AP_ToshibaCAN/AP_ToshibaCAN.cpp

```diff
@@ -297,6 +297,7 @@ void AP_ToshibaCAN::loop()
         if (send_stage == 8) {
             AP_HAL::CANFrame recv_frame;
             while (read_frame(recv_frame, timeout)) {
+#if HAL_WITH_ESC_TELEM
                 // decode rpm and voltage data
```

Comment:
I think you will need to test all these drivers again with ESC telemetry disabled to make sure nothings is broken. Since CAN is only defined on 2Mb boards I really wonder whether it is worth it

Reply:
Should I drop the KDECAN, PicoloCAN, ToshibaCAN and UAVCAN changes?

Reply:
I would suggest so if you get the bulk of the savings from the other changes we discussed